src/ServiceControl/Hosting/Commands/RunCommand.cs

```diff
@@ -23,8 +27,18 @@ public override async Task Execute(HostArguments args, Settings settings)
             hostBuilder.AddServiceControl(settings, endpointConfiguration);
             hostBuilder.AddServiceControlApi();
 
+            var servicePulsePath = Path.Combine(AppContext.BaseDirectory, "platform", "servicepulse", "ServicePulse.dll");
+            var manifestEmbeddedFileProvider = new ManifestEmbeddedFileProvider(Assembly.LoadFrom(servicePulsePath), "wwwroot");
+            var fileProvider = new CompositeFileProvider(hostBuilder.Environment.WebRootFileProvider, manifestEmbeddedFileProvider);
+            var defaultFilesOptions = new DefaultFilesOptions { FileProvider = fileProvider };
+            var staticFileOptions = new StaticFileOptions { FileProvider = fileProvider };
+
```

Comment:
It might be worth extracting all of this into a `.UseServicePulse()` extension to match how we're doing ServiceControl.